src/SmartComponents/Applications/Applications.js

```diff
@@ -1,20 +1,55 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components';
+import { connect } from 'react-redux';
+import { Skeleton, PageHeader, PageHeaderTitle, Main } from '@redhat-cloud-services/frontend-components';
+import FormRender from '@data-driven-forms/react-form-renderer';
+import { formFieldsMapper, layoutMapper } from '@data-driven-forms/pf4-component-mapper';
+import registryDecorator from '@redhat-cloud-services/frontend-components-utilities/files/Registry';
+import { register } from '../../store';
+import reducers  from '../../store/reduces.js';
+import { getSchema } from '../../actions';
 
+const localStorageKey = (appName, user) => `@@settings-${appName}-${user}`;
+
+@registryDecorator()
 class Applications extends Component {
+    constructor(props) {
+        super(props);
+        register(reducers);
+        props.getSchema(props.match.params.id);
+        this.state = {
+            appName: this.props.match.params.id
+        };
+    }
+
+    componentDidMount() {
+        insights.chrome.auth.getUser().then(auth => ({ user: auth.identity.user.username })).then(user => this.setState(user));
+    }
 
     render() {
-        const appName = this.props.match.params.id;
-
-        return (
-            <React.Fragment>
-                <PageHeader>
-                    <PageHeaderTitle title='Applications Settings'/>
-                    <p>{ `Settings for ${ appName }` }</p>
-                </PageHeader>
-            </React.Fragment>
-        );
+        const { appName, user }  = this.state;
+        const { loaded, schema } = this.props;
+        return loaded
+            ? (
+                <React.Fragment>
+                    <PageHeader>
+                        <PageHeaderTitle title='Applications Settings'/>
+                        <p>{ `Settings for ${ appName }` }</p>
+                    </PageHeader>
+                    <Main>
+                        <FormRender
+                            formFieldsMapper={ formFieldsMapper }
+                            layoutMapper={ layoutMapper }
+                            schema={ schema }
+                            onSubmit={ (value) => localStorage.setItem(localStorageKey(appName, user), JSON.stringify(value)) }
```

Comment:
Can you update this function to be dispatched action? Inside this dispatched action you can call `Promise.resolve` with `localStorage.setItem`. This way we will mock storing data trough API and once we have working API we will use it.